src/Controls/src/Core/Compatibility/Handlers/iOS/VisualElementRenderer.cs

```diff
@@ -93,5 +93,22 @@ void OnSizeChanged(object? sender, EventArgs e)
 		{
 			UpdateNativeWidget();
 		}
+
+		void IMauiPlatformView.InvalidateAncestorsMeasuresWhenMovedToWindow()
+		{
+			_invalidateParentWhenMovedToWindow = true;
+		}
+
+		void IMauiPlatformView.InvalidateMeasure(bool isPropagating) => SetNeedsLayout();
+
+		public override void MovedToWindow()
+		{
+			base.MovedToWindow();
+			if (_invalidateParentWhenMovedToWindow)
+			{
+				_invalidateParentWhenMovedToWindow = false;
```

Comment:
Why set to false here? What happens if we remove from the UI and then add it back? This logic will no longer fire? Or is that intentional?